packages/fast-usdc/test/exos/advancer.test.ts

```diff
@@ -336,7 +349,26 @@ test('updates status to OBSERVED if pre-condition checks fail', async t => {
 
   t.deepEqual(inspectLogs(0), [
     'Advancer error:',
-    '"[Error: Unable to parse query params: \\"agoric16kv2g7snfc4q24vg3pjdlnnqgngtjpwtetd2h689nz09lcklvh5s8u37ek\\"]"',
+    '"[Error: query: {} - Must have missing properties [\\"EUD\\"]]"',
+  ]);
+
+  await advancer.handleTransactionEvent({
+    ...MockCctpTxEvidences.AGORIC_NO_PARAMS(
+      encodeAddressHook(
+        'agoric16kv2g7snfc4q24vg3pjdlnnqgngtjpwtetd2h689nz09lcklvh5s8u37ek',
+        {
+          EUD: 'osmo1234',
+          extra: 'value',
+        },
```

Comment:
maybe I shouldn't bother with these style nits, but... any particular reason to split this across 4 lines?

```suggestion
        { EUD: 'osmo1234', extra: 'value' },
```